[2025-07-10 11:11:57 -0400] [50900] [DEBUG] Current configuration:
  config: ./gunicorn.conf.py
  wsgi_app: None
  bind: ['0.0.0.0:5000']
  backlog: 2048
  workers: 4
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/ubuntu/AssociaVal_backend
  daemon: True
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: None
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: debug.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  logconfig_json: None
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f99027b8540>
  on_reload: <function OnReload.on_reload at 0x7f99027b8680>
  when_ready: <function WhenReady.when_ready at 0x7f99027b87c0>
  pre_fork: <function Prefork.pre_fork at 0x7f99027b8900>
  post_fork: <function Postfork.post_fork at 0x7f99027b8a40>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f99027b8b80>
  worker_int: <function WorkerInt.worker_int at 0x7f99027b8cc0>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f99027b8e00>
  pre_exec: <function PreExec.pre_exec at 0x7f99027b8f40>
  pre_request: <function PreRequest.pre_request at 0x7f99027b9080>
  post_request: <function PostRequest.post_request at 0x7f99027b9120>
  child_exit: <function ChildExit.child_exit at 0x7f99027b9260>
  worker_exit: <function WorkerExit.worker_exit at 0x7f99027b93a0>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f99027b94e0>
  on_exit: <function OnExit.on_exit at 0x7f99027b9620>
  ssl_context: <function NewSSLContext.ssl_context at 0x7f99027b9800>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2025-07-10 11:11:57 -0400] [50900] [INFO] Starting gunicorn 21.2.0
[2025-07-10 11:11:57 -0400] [50900] [DEBUG] Arbiter booted
[2025-07-10 11:11:57 -0400] [50900] [INFO] Listening at: http://0.0.0.0:5000 (50900)
[2025-07-10 11:11:57 -0400] [50900] [INFO] Using worker: sync
[2025-07-10 11:11:57 -0400] [50901] [INFO] Booting worker with pid: 50901
[2025-07-10 11:11:57 -0400] [50902] [INFO] Booting worker with pid: 50902
[2025-07-10 11:11:57 -0400] [50903] [INFO] Booting worker with pid: 50903
[2025-07-10 11:11:57 -0400] [50904] [INFO] Booting worker with pid: 50904
[2025-07-10 11:11:57 -0400] [50900] [DEBUG] 4 workers
[2025-07-10 11:12:09 -0400] [50904] [DEBUG] GET /api/associations
[2025-07-10 11:12:15 -0400] [50904] [DEBUG] GET /api/activities
[2025-07-10 11:12:15,044] ERROR in app: Exception on /api/activities [GET]
Traceback (most recent call last):
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/app/routes.py", line 271, in api_activities
    activities = Activity.query.all()
                 ^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/query.py", line 2704, in all
    return self._iter().all()  # type: ignore
           ^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/result.py", line 1774, in all
    return self._allrows()
           ^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/result.py", line 548, in _allrows
    rows = self._fetchall_impl()
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/result.py", line 1681, in _fetchall_impl
    return self._real_result._fetchall_impl()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/result.py", line 2275, in _fetchall_impl
    return list(self.iterator)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/sqlalchemy/orm/loading.py", line 219, in chunks
    fetch = cursor._raw_all_rows()
            ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/result.py", line 541, in _raw_all_rows
    return [make_row(row) for row in rows]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/AssociaVal_backend/venv/lib/python3.11/site-packages/sqlalchemy/engine/result.py", line 541, in <listcomp>
    return [make_row(row) for row in rows]
            ^^^^^^^^^^^^^
  File "lib/sqlalchemy/cyextension/resultproxy.pyx", line 22, in sqlalchemy.cyextension.resultproxy.BaseRow.__init__
  File "lib/sqlalchemy/cyextension/resultproxy.pyx", line 79, in sqlalchemy.cyextension.resultproxy._apply_processors
  File "lib/sqlalchemy/cyextension/processors.pyx", line 40, in sqlalchemy.cyextension.processors.str_to_datetime
ValueError: Invalid isoformat string: '20/07/2025'
[2025-07-10 11:13:28 -0400] [50900] [INFO] Handling signal: term
[2025-07-10 11:13:28 -0400] [50901] [INFO] Worker exiting (pid: 50901)
[2025-07-10 11:13:28 -0400] [50904] [INFO] Worker exiting (pid: 50904)
[2025-07-10 11:13:28 -0400] [50902] [INFO] Worker exiting (pid: 50902)
[2025-07-10 11:13:28 -0400] [50903] [INFO] Worker exiting (pid: 50903)
[2025-07-10 11:13:28 -0400] [50900] [INFO] Shutting down: Master
